### DISCUSSION TITLE
ZuluSCSI V6.4 Rev. 2024a

### DISCUSSION THREAD

**Discussion by IanBullard** [ZuluSCSI/ZuluSCSI-firmware#442] [Q&A] [answered]
I purchased a ZuluSCSI V6.4 Rev. 2024a, and I'm confused.  [The manual](https://github.com/ZuluSCSI/ZuluSCSI-firmware/wiki/ZuluSCSI-Manual) makes no mention of the product.  The firmware releases make no mention of the board.

**Comment by MeTalOneOEight** [comment, marked as answer]
You purchased this:
https://github.com/rabbitholecomputing/ZuluSCSI-V6-firmware
